src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -531,13 +531,17 @@ private RecommendedSymbols GetSymbolsOffOfExpressionInConstantPattern(Expression
             if (boundSymbol is not INamespaceOrTypeSymbol namespaceOrType)
                 return default;
 
+            var containingType = _context.SemanticModel.GetEnclosingNamedType(_context.Position, _cancellationToken);
+            if (containingType == null)
+                return default;
```

Comment:
If you're in a top-level statement case (i.e. implicit main) does this return the made-up Program class or return null?

Reply:
I'm guessing it returns the made-up Program class, because the tests for this feature are implemented using top level statements.

Reply:
the to plevel program class.